deps/unwind.mk

```diff
@@ -30,10 +30,6 @@ endif
 
 define LIBUNWIND_INSTALL
```

Comment:
can delete this entire block and just use `MAKE_INSTALL` as the argument to staged-install now